### ISSUE TITLE
Tested with MobileNetV2 SSD-Lite model

### ISSUE THREAD

**Issue by HKLee2040** [jakc4103/DFQ#17]
While I have the following command:
python main_ssd.py --quantize --correction --log --bits_bias 16

It shows me the warning message:
Warning: <class 'utils.quantize.QuantNConv2d'> layer before first batch norm layer detected. The calculated value range might be off.
Warning: <class 'utils.quantize.QuantNConv2d'> layer before first batch norm layer detected. The calculated value range might be off.
...

and the mAP is always below than your result, i.e., 0.6729297

Does the warning message lead to accuracy drop?

**Comment by jakc4103** [comment]
I quantize all inputs(detection layer output) of torch.cat(the final op), and there is no BN between Conv(detection layer) and torch.cat. 
Say we have a Conv called layer_a, the value range to quantize output of layer_a is "estimated" by convole the BN stats before layer_a with layer_a's weight. That's why I print the warning message.

The mAP might be different cause I tweak some of implementations in set_quant_minmax and did not completely re-run all experiments; and using fake_quant in CPU or GPU may lead to different results. 
Personally I recommend to use true int8 inference engine to test the results.

**Comment by HKLee2040** [comment]
I'm sorry!  I forgot that I modified the code.
The result is very close to the public data.
